### PR TITLE
Fix kubeconfig injection into containers

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Fix kubeconfig injection into containers

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/21147

### Is it tested? How?

1. Start a workspace.
2. Search for `~/.kube/config` file.
See: the file is present in the user directory:

You can check it with https://che-eclipse-che.apps.cluster-48vcj.48vcj.sandbox281.opentlc.com/

### CODE REVIEW

**Comment by musienko-maxim** [comment]
All e2e functionality was checked and worked as expected. Merge, please

**Review by ibuziuk — APPROVED** [review]

File: run/prepare-local-run.sh [hunk 1]
```diff
@@ -57,8 +62,10 @@ if [[ ! -z "$GATEWAY" &&
 
     # rollout Dex deployment
     echo 'Rolling out Dex deployment...'
-    oc patch deployment/dex --patch "{\"spec\":{\"replicas\":0}}" -n dex
-    oc patch deployment/dex --patch "{\"spec\":{\"replicas\":1}}" -n dex
+    kubectl patch deployment/dex --patch "{\"spec\":{\"replicas\":0}}" -n dex
+    echo 'Waiting 5 seconds to operator shut down...'
```
Comment (akurinnoy) [inline run/prepare-local-run.sh:66]: wrong copy-paste: ~~operator~~dex

File: run/revert-local-run.sh [hunk 2]
```diff
@@ -57,8 +63,10 @@ if kubectl get configMaps che-gateway-config-oauth-proxy -o jsonpath="{.data}" -
 
   # rollout che-server deployment
   echo 'Rolling out che-gateway deployment...'
-  oc patch deployment/che-gateway --patch "{\"spec\":{\"replicas\":0}}" -n $CHE_NAMESPACE
-  oc patch deployment/che-gateway --patch "{\"spec\":{\"replicas\":1}}" -n $CHE_NAMESPACE
+  kubectl patch deployment/che-gateway --patch "{\"spec\":{\"replicas\":0}}" -n $CHE_NAMESPACE
+  echo 'Waiting 5 seconds to operator shut down...'
```
Comment (akurinnoy) [inline run/revert-local-run.sh:67]: ~~operator~~dex

**Review by akurinnoy — APPROVED** [review]
LGTM

File: run/prepare-local-run.sh (hunk 1 above)
Comment (olexii4) [reply run/prepare-local-run.sh]: I have fixed it.

File: run/revert-local-run.sh (hunk 2 above)
Comment (olexii4) [reply run/revert-local-run.sh:67]: I have fixed it.

**Pull request merged**